src/components/Office/ShipmentForm/ShipmentForm.jsx

```diff
@@ -465,6 +486,7 @@ const ShipmentForm = (props) => {
               onClose={() => setIsAddressChangeModalOpen(false)}
               deliveryAddressUpdate={mtoShipment?.deliveryAddressUpdate}
               shipmentType={mtoShipment?.shipmentType}
+              // TODO: onSubmit
```

Comment:
Here's where we'll pass it that submit handler in

```suggestion
              onSubmit={handleSubmitShipmentAddressUpdateReview}
```